.github/workflows/test.yml

```
@@ -128,7 +128,7 @@ jobs:
       matrix:
         python-version: ["3.10", "3.11", "3.12", "3.13"]
         airflow-version: ["2.6", "2.7", "2.8", "2.9", "2.10", "2.11", "3.0", "3.1"]
-        dbt-version: ["1.10"]
+        dbt-version: ["1.10", "1.11"]
```

```suggestion
        dbt-version: [ "1.11" ]
```
@pankajkoti I don't think we should run Cosmos integration tests for both versions of dbt Core. Previously, we were only running against the latest minor. It is expensive to run them. We currently run 4x8 integration test jobs (32 in total). By adding this, we'd be running 64 jobs, without any clear benefit.

The main part of Cosmos that is sensitive to the dbt version is the async implementation - and for those we should certainly test against as many dbt versions as possible.

If we notice that things stop working with a specific dbt 1.10 version, we can enable testing against those versions, but that doesn't justify the cost for all tests.